### DISCUSSION TITLE
Do I need pages folder in document driver mode?

### DISCUSSION THREAD

**Discussion by victor-ponamariov** [nuxt/content#1810] [Q&A]
I'm not sure I quite understand the mode.

E.g., previously without that mode I have such pages as home (list of articles), newsletter, about me, and so on.

Should Inow completely remove pages folder and move everything in /content/, embedding components for these pages via MDC?

Because otherwise I believe Nuxt tries to query unexisting index page in /cotent folder, even though the index.vue page works if it exists in /pages folder

**Comment by Zerro97** [comment]
I actually had the same [question](https://github.com/nuxt/content/discussions/1904) and didn't know someone actually posted it beforehand.. 

I think with document driven enabled, it automatically registers [NuxtPlugin](https://github.com/nuxt/content/blob/main/src/runtime/plugins/documentDriven.ts) which queries page data from `/api/_content`. This querying page data happens during any page load.

If the application is entirely based on markdown, it definitely make sense to use document-driven feature, and do not need pages folder at all.

If some parts of the application is not based on markdown, well, I think it's still alright to use document driven mode (despite the error messages...).
